src/PrestaShopBundle/Controller/Admin/Improve/Payment/PaymentMethodsController.php

```diff
@@ -26,16 +26,24 @@
 
 namespace PrestaShopBundle\Controller\Admin\Improve\Payment;
 
-use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShop\PrestaShop\Adapter\Presenter\Module\PaymentModulesPresenter;
+use PrestaShopBundle\Controller\Admin\PrestaShopAdminController;
 use PrestaShopBundle\Security\Attribute\AdminSecurity;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
  * Class PaymentMethodsController is responsible for 'Improve > Payment > Payment Methods' page.
  */
-class PaymentMethodsController extends FrameworkBundleAdminController
+class PaymentMethodsController extends PrestaShopAdminController
 {
+    public static function getSubscribedServices(): array
```

Comment:
You could have injected it in the method directly (to avoid updating the PHPStan exceptions) but it's fine this way as well

Reply:
I prefer too. In fact, I've first do this way because we use this presenter only in singleShopContext.